### PR TITLE
AFrame 1.0.4 --> 1.0.3 + introduce delay to NAF

### PR DESCRIPTION
**Pull request by nazariyv**

### CODE REVIEW

File: module/nft.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export const getNfts = (address) => {
+  let data = null;
+
+  const xhr = new XMLHttpRequest();
```
Comment (microchipgnu) [inline module/nft.js:4]: Use fetch (https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API). It's widely supported.

File: init.js [hunk 2]
```diff
@@ -15,12 +15,22 @@ const _clientConnected = (event) => {
...
   // send user's data to connecting client
-  NAF.connection.sendDataGuaranteed(
+  // ! add a small delay so that we have the naf-clientNetworkId
+  // TODO: find a better way to do this, w/o the timeout
+  setTimeout(() => NAF.connection.sendDataGuaranteed(
     clientId,
     "account",
     clientMessage
-  );
+  ), 1000);
```
Comment (laughinglemur) [inline init.js:33]: Making a note here that this is the major change in this PR

**Review by laughinglemur — APPROVED** [review]
Good to merge after modifying retries to be separated by 200 ms each.

File: module/messages.js [hunk 3]
```diff
@@ -34,7 +35,11 @@ export const messages = {
 
     const naf = document.querySelector(`#naf-${clientNetworkId}`);
     if (naf == null) {
-      return
+      if (counter + 1 >= 5) {
```
Comment (nazariyv) [inline module/messages.js:38]: this is a major change as well. recursive calls if it is still null

**Pull request merged**